cmd/credential/password/password.go

```diff
@@ -49,21 +52,27 @@ const (
 	FlagNamePasswordCount            = "count"
 )
 
+// [Q]: don't get why did you use this interface.
+// You didn't implement this interface in your code.
 type PasswordGenerator interface {
 	Write(io.Writer) error
 	Generate(int) string
 }
 
+// [Q]: don't get why did you use this struct.
```

Comment:
I was trying not to use global variables.. but looks like for this simple utility and for our current scope, i can let go of this struct.so removing it.